Dokumentationen/test/test_cases.adoc

```diff
@@ -115,3 +115,110 @@ a| * Valides Datum, valide Uhrzeit und valider Studiengang
 
 ==== TC10 - Statistik wird richtig angezeigt 
 
+=== Test Cases für Use Case "Dozentenwoche anzeigen"
+
+==== TC11 - Dozentenwoche für einen Dozenten wird korrekt angezeigt
+
+|===
+
+|*Beschreibung* |Der Admin möchte sich die Dozentenwoche für eine bestimmte KW und einen bestimmten Dozenten ausgeben lassen.
+
+|*Vorbedingungen*
+a| * Admin ist angemeldet  
+* Admin befindet sich auf der Website für die Dozentenwoche
+
+|*Nachbedingung*
+a| * Die entsprechend gefilterte Dozentenwoche wird angezeigt
+
+|*Benötigte Daten*
+a| * Valide KW, valider Dozentenname
+* die von dem Dozenten gebuchten Experimente mit dem jeweiligen Datum, Uhrzeit und Studiengang
+
+|===
+
+
+==== TC12 - Dozentenwoche für mehrere Dozenten wird korrekt angezeigt
```

Comment:
Diesen TC gibt es glaube ich nicht, da es unser System nicht unterstützt. Weil wir können nur einen Dozenten auswählen und nicht mehrere. So ist es mal bis jetzt gedacht oder ?

Reply:
Ich dachte schon, dass der Admin sich alle in einem gesammelten Blatt ausgeben kann.

Reply:
Also ich habe mir jetzt noch mal den UC angeschaut und es geht immer nur um Dozent und KW und nicht um Dozenten oder KWochen. Und beim Wireframe ist es auch ein dropdown Menü, somit kann man ja gar nicht mehr als einen auswählen ansonsten bräuchten wir checkboxen

Reply:
Okay, dann lass ich den TC weg

Reply:
done